.github/workflows/full-bundle-tests.yaml

```diff
@@ -9,6 +9,18 @@ on:
       bundle-source:
```

Comment:
would it make sense to use only `bundle-source` input and figure out the `bundle-test-path` from that?
It feels redundant to do `test folder: tests-bundle/1.8` and `channel: --channel=1.8/stable`

Reply:
I think leveraging the script makes sense for the bundle path.
about Juju and uk8s, I can see a case where we want to specify those versions ourselves, so I think it's ok to keep them as inputs.

Reply:
Agreed. This way, other workflows that need to use this one (`workflow_call`) won't have to specify the `bundle-test-path` input too. Just pushed 2 commits for this.